nixos/modules/system/boot/uvesafb.nix

```
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.boot.uvesafb;
+in {
+  options = {
+    boot.uvesafb = {
+      enable = lib.mkEnableOption (lib.mdDoc "uvesafb");
+
+      gfx-mode = lib.mkOption {
+        type = lib.types.str;
+
+        default = "1024x768-32";
+
+        description = lib.mdDoc "Screen resolution";
+      };
+
+      v86d.package = lib.mkOption {
+        type = lib.types.package;
+
+        description = lib.mdDoc "Which v86d package to use with uvesafb";
+
+        defaultText = "config.boot.kernelPackages.v86d.overrideAttrs (old: {
+          hardeningDisable = [ \"all\" ];
+        })";
+
```

```suggestion
        description = lib.mdDoc "Which v86d package to use with uvesafb";
        defaultText = "config.boot.kernelPackages.v86d.overrideAttrs (old: {
          hardeningDisable = [ \"all\" ];
        })";
```

Also, disable hardening here looks dangerous...

---

It crashes without this.